centaur/src/main/resources/standardTestCases/wdl_draft3/empty/empty.wdl

```diff
@@ -0,0 +1,3 @@
+version draft-3
+
+workflow empty {}
```

Comment:
1) Until centaur prints out the workflow ids for each test, I like when the workflow tests match the workflow names (as much as possible). Helps when grepping through the logs, trying to figure out what failed workflow matches "should successfully run find_me_if_you_can_muhahaha *** FAILED ***".
2) I prefer files end with newlines.